src/processor/reconnection-graph.service.ts

```diff
@@ -367,4 +416,14 @@ export class ReconnectionGraphService {
     };
     return dsnpKeys;
   }
+
+  async processChainEvents(promises: Promise<ParsedEventResult>[], dsnpUserId: MessageSourceId): Promise<void> {
+    // loop over promises and wait for all to resolve
+    for (const promise of promises) {
+      const [event, eventMap] = await promise;
```

Comment:
here, `event` will always be undefined unless you pass a target event to the extrinsic creation. Note the current tooling only allow for target events to be returned if the call was created with `createExtrinsic`, not `createExtrinsicCall`